index.html

```diff
@@ -276,12 +279,9 @@ <h1 id = "contact-me">
                         Contact me
```

Comment:
- [ ] Kindly ensure your footer section matches the design given kindly check the comparison below:


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/178119743-a35e303f-112c-49b4-9cde-b1efe336c255.png) |  ![image](https://user-images.githubusercontent.com/61408860/178119750-cd723bf4-1fa0-4e54-a6fa-23740775ca30.png)